### PR TITLE
Android ui fix

### PR DESCRIPTION
**Pull request by SambaDialloB**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -602,70 +641,53 @@ export default class App extends Component {
                 {(function() {
                   let rows = [];
                   for (let i = 0; i < item.emojiCount; i++) {
-                    console.log(item.emojiCount);
+                    //console.log(item.emojiCount);
+                    let emoji;
```
Comment (lovdeep7) [inline App.js]: let's be a little more descriptive with our Emoji variable names; we've been calling everything `emoji`

File: src/components/EmojiBar/EmojiBar.js [hunk 2]
```diff
@@ -179,7 +179,7 @@ export default class AnimationScreen extends Component {
     // the margin top the box is 100
     // and plus the height of toolbar and the status bar
     // so the range we check is about 150 -> 450
-    if ((Platform.OS === 'android' && gestureState.y0 + gestureState.dy >= 450 && gestureState.y0 + gestureState.dy <= 600)
+    if ((Platform.OS === 'android' && gestureState.y0 + gestureState.dy >= 600 && gestureState.y0 + gestureState.dy <= 1000)
```
Comment (lovdeep7) [inline src/components/EmojiBar/EmojiBar.js]: keep this between 450 and 600 for now until we figure out how to make this responsive

File: src/components/ModalAppUpdate.js [hunk 3]
```diff
@@ -90,86 +95,86 @@ export default class ModalAppUpdate extends Component {
...
                 <View style={styles.username}>
-                    <TextInput 
+                    <TextInput
                       style={{flex: 1}}
-                      type="TextInput" 
-                      name="myTextInput" 
-                      placeholder='Change your username' 
+                      type="TextInput"
+                      name="myTextInput"
+                      placeholder= {this.props.currentUsername}
```
Comment (lovdeep7) [inline src/components/ModalAppUpdate.js]: if they haven't set a username the default should be something like as previous (Change your username)

File: src/components/EmojiBar/EmojiBar.js (hunk 2 above)
Comment (SambaDialloB) [reply src/components/EmojiBar/EmojiBar.js]: I used 600 because I can then test it on my phone

File: src/components/ModalAppUpdate.js (hunk 3 above)
Comment (SambaDialloB) [reply src/components/ModalAppUpdate.js]: This is in the update modal, when they open it for the first time the initial modal will show

File: App.js (hunk 1 above)
Comment (SambaDialloB) [reply App.js]: Removed alot of the emoji stuff in my mergeChannels branch I have locally

**Comment by SambaDialloB** [comment]
I have this plus, the mergeChannels branch to push up, now Working on making it responsive now

File: src/components/EmojiBar/EmojiBar.js (hunk 2 above)
Comment (lovdeep7) [reply src/components/EmojiBar/EmojiBar.js]: all good, i fixed this issue in https://github.com/PubNubDevelopers/PubMoji/pull/22 

after merging this all devices should work

File: App.js (hunk 1 above)
Comment (lovdeep7) [reply App.js]: kk if you can make a little diagram of the payload after you've merged all the channels it'll be sick

**Pull request closed without merging**